edenai_apis/tests/utils/test_files.py

```
@@ -0,0 +1,181 @@
+            extra="value",
+        )
+        assert info.metadata["duration"] == 10.0
+        assert info.metadata["extra"] == "value"
+
+
+class TestFileWrapperInit:
+    @pytest.mark.unit
+    def test_basic_init(self):
+        info = FileInfo(file_size=100, file_mimetype="text/plain", file_extension="txt")
+        wrapper = FileWrapper(
+            file_path="/tmp/file.txt",
```

Ruff (0.15.10)

[error] Probable insecure usage of temporary file or directory: "/tmp/file.txt"

(S108)